frontend/test/metabase/scenarios/visualizations/line_chart.cy.spec.js

```diff
@@ -437,5 +437,5 @@ function showTooltipForFirstCircleInSeries(series_index) {
     .as("firstSeries")
     .find("circle")
     .first()
-    .realHover();
+    .trigger("mousemove", { force: true });
```

Comment:
Looks like this is some leakage from https://github.com/metabase/metabase/pull/17946 -- might want to remove it to avoid a merge conflict